BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1944,6 +1991,22 @@ void EndCruise()
         {
             MissileState = MissileStates.PostThrust;
 
+            if (BDArmorySettings.MISSILE_FUEL_DEDUCTION)
+            {
+                float deduction = (sustainerMass < 0) ? 0 : sustainerMass;
+                if (BDArmorySettings.DEBUG_MISSILES) Debug.Log($"[BDArmory.Missile.FuelDeduction]: if FuelDeduction on Cruise is working this {part.mass} changed.");
```

Comment:
As with the logging/debug check? on ln 1849, this is unnecessary, as everything should already be configured and outputting the right numbers; there's nothing that would change fuel amount post Start() that would result in erroneous deduction

Reply:
This bit was a part that i forgot to remove, i used when i was testing.